assets/src/blocks/CarouselHeader/CarouselHeaderEditor.js

```diff
@@ -16,7 +16,7 @@ export const CarouselHeaderEditor = ({ setAttributes, attributes }) => {
   const slidesRef = useRef([]);
   const slidesWithImages = useCarouselHeaderImages(slides);
 
-  const { currentSlide, goToSlide, goToNextSlide, goToPrevSlide } = useSlides(slidesRef, slides.length);
+  const { currentSlide, goToSlide, goToNextSlide, goToPrevSlide } = useSlides(slidesRef, slides.length - 1);
```

Comment:
This was actually broken, meant that in the backend we couldn't go from last slide back to the first one using the arrows 😬